WooCommerce/Classes/ViewRelated/Products/Media/Product+Media.swift

```diff
@@ -18,6 +18,13 @@ extension Product {
     }
 }
 
+extension ProductListItem {
+    /// Returns the URL of the first image, if available. Otherwise, nil is returned.
+    var imageURL: URL? {
+        images.first?.imageURL
```

Comment:
nit: perhaps this can be moved to the conversion logic from `StorageProduct` to the readonly type in order details, if the use case doesn't need to know all the images?